src/libraries/System.Numerics.Tensors/ref/System.Numerics.Tensors.csproj

```diff
@@ -8,7 +8,7 @@
     <Compile Include="System.Numerics.Tensors.cs" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'">
+  <ItemGroup Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net7.0'))">
     <Compile Include="System.Numerics.Tensors.netcore.cs" />
   </ItemGroup>
```

Comment:
Good point, since this is more in the style realm, I'll let @michaelgsharp / @stephentoub / @tannergooding consider a rename here. https://github.com/dotnet/runtime/issues/92219

Reply:
I'm having a hard time thinking of a better name. The suffix is really about the surface area it's targeting / being constrained to, which is the netstandard surface area. The fact that it's also being used in a build for a .NET Core TFM doesn't really matter, IMO.

Reply:
Let's address this request for GA. This is blocking the RC2 snap, so I merged it as-is.